examples/main/src/app.js

```diff
@@ -91,18 +89,18 @@ class App extends PureComponent {
     this.setState({settings});
   }
 
+  _onHover(info) {
+    this.refs.infoPanel.onItemHovered(info);
```

Comment:
Do we need to use refs here? Doesn't look very "reactive". Are we just trying to avoid setting app state or is there a reason why it needs to be done like this? Ether way, add a comment?